### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by AnsMann**

### CODE REVIEW

File: src/app/DeleteModalDialogue.js [hunk 1]
```diff
@@ -60,11 +60,9 @@ const StyledButton = styled.button`
 `
 
 export function DeleteModalDialogue({
```
Comment (leojclarke) [inline src/app/DeleteModalDialogue.js]: "close" would be a better term than delete

File: src/app/DeleteModalDialogue.js [hunk 2]
```diff
@@ -60,11 +60,9 @@ const StyledButton = styled.button`
...
-  entryId,
   entryDate,
-  history,
   onDeleteConfirmation,
-  onDeleteAbort,
+  resetDeleteModal,
```
Comment (leojclarke) [inline src/app/DeleteModalDialogue.js]: "resetCloseModalState"...? Nit picking here...

**Review by leojclarke — COMMENTED** [review]
Great job! Cleaver way of really reducing the size of App.js.

File: src/app/DeleteModalDialogue.js (hunk 1 above)
Comment (AnsMann) [reply src/app/DeleteModalDialogue.js]: i changed the naming to make it more clear

File: src/app/DeleteModalDialogue.js (hunk 2 above)
Comment (AnsMann) [reply src/app/DeleteModalDialogue.js]: i changed that

**Pull request merged**